[2024-06-03T21:59:10.296+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T21:59:10.315+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-03T21:59:10.321+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-03T21:59:10.322+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T21:59:10.333+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-04-01 00:00:00+00:00
[2024-06-03T21:59:10.340+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=27507) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T21:59:10.342+0000] {standard_task_runner.py:63} INFO - Started process 27509 to run task
[2024-06-03T21:59:10.342+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpj2v3rzrp']
[2024-06-03T21:59:10.343+0000] {standard_task_runner.py:91} INFO - Job 119: Subtask blunders_etl
[2024-06-03T21:59:10.377+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T21:59:10.444+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-06-03T21:59:10.445+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T21:59:10.446+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T21:59:10.452+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:02:23.912+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:02:23.912+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:02:23.920+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240603T215910, end_date=20240603T220223
[2024-06-03T22:02:23.960+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:02:23.973+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:02:23.975+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:26:12.618+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:26:12.633+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-08T19:26:12.639+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-08T19:26:12.639+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:26:12.649+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-04-01 00:00:00+00:00
[2024-06-08T19:26:12.656+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=27915) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:26:12.658+0000] {standard_task_runner.py:63} INFO - Started process 27917 to run task
[2024-06-08T19:26:12.657+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpja35pvox']
[2024-06-08T19:26:12.659+0000] {standard_task_runner.py:91} INFO - Job 295: Subtask blunders_etl
[2024-06-08T19:26:12.692+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:26:12.754+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-06-08T19:26:12.756+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:26:12.757+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:26:12.763+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T19:29:15.094+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T19:29:15.095+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T19:29:15.102+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240608T192612, end_date=20240608T192915
[2024-06-08T19:29:15.150+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T19:29:15.160+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T19:29:15.161+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:24:05.130+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:24:05.146+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-07-08T01:24:05.151+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-07-08T01:24:05.152+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:24:05.160+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-04-01 00:00:00+00:00
[2024-07-08T01:24:05.167+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=28741) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:24:05.169+0000] {standard_task_runner.py:63} INFO - Started process 28743 to run task
[2024-07-08T01:24:05.169+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '665', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp1xa53x9r']
[2024-07-08T01:24:05.170+0000] {standard_task_runner.py:91} INFO - Job 665: Subtask blunders_etl
[2024-07-08T01:24:05.203+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:24:05.271+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-07-08T01:24:05.272+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:24:05.273+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:24:05.279+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T01:27:05.534+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T01:27:05.534+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T01:27:05.542+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240708T012405, end_date=20240708T012705
[2024-07-08T01:27:05.591+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T01:27:05.610+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T01:27:05.611+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
